app/api/events.py

```diff
@@ -31,18 +31,63 @@ class Meta:
 
     id = fields.Str(dump_only=True)
     identifier = fields.Str(dump_only=True)
-    name = fields.Str()
-    event_url = fields.Url()
-    starts_at = fields.DateTime(required=True)
-    ends_at = fields.DateTime(required=True)
-    tickets = Relationship(attribute='ticket',
+    name = fields.Str(required=True)
+    email = fields.Str(required=True)
+    event_url = fields.Url(dump_only=True)
```

Comment:
@LuD1161 Please change this back to fields.Str(), you said this was resolved in the same comment I made earlier